pkg/kn/commands/route/list_flags_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or im
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"testing"
+
+	"github.com/knative/client/pkg/kn/commands"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func TestRoutListFlags(t *testing.T) {
+	testObject := createMockRouteMeta("foo")
+	knParams := &commands.KnParams{}
+	cmd, _, buf := commands.CreateTestKnCommand(NewRouteCommand(knParams), knParams)
+	routeListFlags := NewRouteListFlags()
+	routeListFlags.AddFlags(cmd)
+	printer, err := routeListFlags.ToPrinter()
+	// check if printer is compatible
+	if genericclioptions.IsNoCompatiblePrinterError(err) {
+		t.Fatalf("Expected to match human readable printer.")
+	}
+	if err != nil {
+		t.Fatalf("Failed to find a proper printer.")
+	}
+	// check if its able to printer can print the object
```

Comment:
[Nit] part of why I hate ( 😀 ) comment for somewhat obvious code. They get out of date and can be more confusing than code or English hard to parse (as above): "check if its able to printer can print the object" what? :)

Reply:
typo!
removed obvious code comments!!